rfq/order.go

```diff
@@ -125,13 +127,20 @@ type AssetSalePolicy struct {
 	// wants us to produce NoOp HTLCs.
 	NoOpHTLCs bool
 
+	auxChanNegotiator *tapfeatures.AuxChannelNegotiator
+
+	peer route.Vertex
```

Comment:
(This can be axed as well if it proves an appropriate 'use no-op HTLCs' check can be performed earlier.)